scripts/output/comparison/notebook_templates/AriadneComparison.Rmd

```diff
@@ -1982,6 +1982,107 @@ ggsave(plot = p.ind.nonen,
        width=20, height=10, units="cm")
 ```
 
+## FE emission intensity
+
+```{r, echo=FALSE, message=FALSE, warning=FALSE}
+
+df <-  df %>%
+                 calc_addVariable(
+                       "`FE|Emission|Intensity|Fuel`"  =  "(1e9*(`Emi|CO2|Energy|Demand|+|Industry` +
+ `Emi|CO2|Energy|Demand|+|Buildings` + `Emi|CO2|Energy|Demand|+|Transport`))/((`FE|Gases|+|Hydrogen` +
+ `FE|Gases|+|Fossil` + `FE|Solids|+|Fossil` + `FE|Solids|+|Biomass` + `FE|Liquids|+|Hydrogen` +
+`FE|Liquids|+|Fossil` + `FE|Liquids|+|Biomass`)*277777777.78)", ## convert MtCO2 in kgCO2 and EJ into MWh
+                       "`FE|Emission|Intensity|Electricity`"  =  "(1e9*`Emi|CO2|Gross|Energy|Supply|+|Electricity`)/(`FE|+|Electricity`*277777777.78)",
```

Comment:
This needs to be `` Emi|CO2|Gross|Energy|Supply|+|Electricity / SE|Electricity`` because SE Electricity can also go to SE|Hydrogen and not only to FE|Electricity.